### PR TITLE
#7414 correct condition and exit

### PR DESCRIPTION
**Pull request by KILFaT**

### CODE REVIEW

File: maxscale_jobs/adding_new_line_in_file.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - string:
+          name: test_set
+      - choice:
+          name: file
+          choices:
+            - $HOME/run_test_branches.list
+            - $HOME/on_push_maxscale_branches.list
+      - string:
+          name: repo
+          default: "https://github.com/mariadb-corporation/MaxScale.git"
```
Comment (timofeyturenko) [inline maxscale_jobs/adding_new_line_in_file.yaml]: there is yaml with default Maxscale repository, I guess we should not hardcode it here

File: maxscale_jobs/adding_new_line_in_file.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    builders:
+      - shell: |
+          if [[ $(git ls-remote --heads $repo $branch) ]]; then
```
Comment (zmm) [inline maxscale_jobs/adding_new_line_in_file.yaml]: Please add check for branch existing in file.

File: maxscale_jobs/adding_new_line_in_file.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            echo "$branch exist"
+            echo "$branch|$test_set" >> $file
+            echo "$branch|$test_set added to $file"
+          else
```
Comment (zmm) [inline maxscale_jobs/adding_new_line_in_file.yaml]: Rewrite condition to following way:
if error then echo and exit.

File: maxscale_jobs/adding_new_line_in_file.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+- job:
+    name: adding_new_line_in_file
```
Comment (zmm) [inline maxscale_jobs/adding_new_line_in_file.yaml]: Please rename job and file to update_branches_file

**Pull request closed without merging**